foreach.inc

```diff
@@ -1021,7 +1021,11 @@ stock Iter_ScriptInit()
 		Iter_CreateVehicle
 	*/
 
+#if defined _INC_open_mp
+	stock Iter_CreateVehicle(modelid, Float:x, Float:y, Float:z, Float:angle, color1, color2, respawn_delay, bool: addsiren = false)
```

Comment:
`bool: addsiren` - excess space

Reply:
Yes, it's better

Reply:
Can you remove that excess space symbol?

Reply:
Done